autocertkit/utils.py

```diff
@@ -69,6 +69,9 @@
 # XAPI States
 XAPI_RUNNING_STATE = "Running"
 
+# Multicast
+MULTICAST_IP = "226.94.1.1"
```

Comment:
Sorry moving to utils.py isn't the solution that I asked for. My question is whether it is okay to use 'hard-coded' IP in the code. Probably better using a configurable value?
If it is allowed being hard-coded, then I think network_tests.py can be more appropriate place.